packages/entities/entities-shared/src/components/entity-base-table/EntityBaseTable.vue

```diff
@@ -286,6 +288,7 @@ const headers = computed<Array<InternalHeader>>(() => {
       label: field.label ?? key,
       key,
       sortable: field.sortable ?? false,
+      hidable: true,
```

Comment:
This needs to be updated to not be applied to headers with `actions` as the key (don't allow hiding the action menus)

Reply:
I think it's a `no-op` since `actions` item is already pushed into `arr` as a separate entry [here](https://github.com/Kong/public-ui-components/blob/main/packages/entities/entities-shared/src/components/entity-base-table/EntityBaseTable.vue#L292-L297)